### ISSUE TITLE
VipsForeignSavePngFile does not support argument Q

### ISSUE THREAD

**Issue by in-in** [libvips/pyvips#153]
The documentation says that `pngsave` supports the `Q` argument.
![200191579466733](https://user-images.githubusercontent.com/8797432/72688404-f6419080-3b17-11ea-96a6-336b5afff20e.png)
https://libvips.github.io/pyvips/vimage.html#pyvips.Image.pngsave

But my script crashes with the error: "_VipsForeignSavePngFile does not support argument Q_"

<details>
  <summary>Traceback</summary>

  <pre>
Traceback (most recent call last):
  File "/home/dmin/bin/image_optim", line 22, in <module>
    optim(entry.name, image)
  File "/home/dmin/bin/image_optim", line 15, in optim
    image.write_to_file(name[0] + ".min." + name[1], Q=80)
  File "/home/dmin/.local/lib/python3.6/site-packages/pyvips/vimage.py", line 501, in write_to_file
    string_options=options, **kwargs)
  File "/home/dmin/.local/lib/python3.6/site-packages/pyvips/voperation.py", line 208, in call
    '{1}'.format(operation_name, name))
pyvips.error.Error: VipsForeignSavePngFile does not support argument Q
  </pre>
</details>

my script:
```python
#!/usr/bin/env python3

# https://libvips.github.io/pyvips/

# PREREQUISITES: libvips42, pyvips

from os import scandir, getcwd
from pyvips import Image

cwd = getcwd()


def optim(filename, image):
    name = filename.rsplit('.', maxsplit=1)
    image.write_to_file(name[0] + ".min." + name[1], Q=80)


for entry in scandir(cwd):
    if entry.is_file() and entry.name.lower().endswith(
        ('.png', '.jpg', '.jpeg', 'webp')):
        image = Image.new_from_file(entry.path)
        optim(entry.name, image)
```

I use _pyvips 2.1.8_

**Comment by jcupitt** [comment]
Hello @in-in, 

The Q option was added in libvips 8.7, so i guess you have an older libvips. You'll also need to have libimagequant enabled if you want to generate 8-bit pallettized PNG images.

**Comment by in-in** [comment]
Yes, you're right.
In my system(Linux Mint 19.3), I use the `libvips42` version _8.4.5-1build1_. 
I'm closing this issue because there's no problem in `pyvips`. 
Thank you so much for the explanation.

**Comment by jeremyEudaric** [comment]
hello l got the same error :
import pyvips as pv
pv = pv.Image.new_from_file("/Users/eudari01/..../test/1.jpg", level=0, access="sequential")
pv.tiffsave('1.tif')
Error: VipsForeignLoadJpegFile does not support optional argument level
could you help me ?

**Comment by jcupitt** [comment]
Please don't ask the same question in several place, it just makes more work for me. Open a new issue for a new question, and explain what you are trying to do.

**Comment by jeremyEudaric** [comment]
Sorry 🙏🏽

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
